tests/unit/tree/request_node/test_run.py

```diff
@@ -43,6 +43,7 @@ def test_calls_request(self, mock_session, mock_time_sleep):
             "response": mock_session().request(),
             "tests_results": [],
             "no_failure": True,
+            "request_node_name": "foo",
```

Comment:
@hebertjulio maybe it would be nice to change this line, to have EndpointNode with another name.

https://github.com/scanapi/scanapi/pull/390/files#diff-8394851ab4aa52ae1478a0a99c89a680eb2166df3e60a30f5f27643f4c8b3785R26

Something like:

```python
request = RequestNode(
    {"path": "http://foo.com", "name": "foo"},
    endpoint=EndpointNode(
        {"name": "endpoint_name", "requests": [{}], "delay": 1}
    ),
)
```

Because then, we know that the `"foo"` here comes certainly from request and not from the endpoint. What do you think?

Reply:
i think great suggestion @camilamaia , and i can change request name too from foo to request_name to keep same pattern.